<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>713</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Menu</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="banner">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>39</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <widget class="QMenu" name="menuModulos">
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Modulos</string>
    </property>
    <addaction name="actionClientes"/>
    <addaction name="actionProductos"/>
    <addaction name="actionVentas"/>
   </widget>
   <addaction name="menuModulos"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionClientes">
   <property name="text">
    <string>Clientes</string>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
  </action>
  <action name="actionProductos">
   <property name="text">
    <string>Productos</string>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
  </action>
  <action name="actionVentas">
   <property name="text">
    <string>Ventas</string>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
